docs/guides/interactive-components.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: "Interactive Components"
+lang: en
+---
+
+# Interactive Components
+
+[Interactive components](https://api.slack.com/reference/block-kit/interactive-components) are a subset of Block Kit elements that add interactivity to various app surfaces. Interactions on blocks may happen not only in messages but also on [Modals]({{ site.url | append: site.baseurl }}/guides/modals), and [Home tabs]({{ site.url | append: site.baseurl }}/guides/app-home).
```

Comment:
"that add interactivity to various app surfaces." -> "that add interactivity to various [surfaces](https://api.slack.com/surfaces)."